docs/src/prototypes/chatMessageWithPopover/Popover.tsx

```diff
@@ -28,6 +28,14 @@ class Popover extends React.Component<PopoverProps, PopoverState> {
     this.changeFocusState(shouldPreserveFocusState)
   }
 
+  getToolbarBehavior: Accessibility = (props: any) => {
+    const behavior = toolbarBehavior(props)
+    behavior.focusZone.props.defaultTabbableElement = (root: HTMLElement): HTMLElement => {
+      return root.querySelector('[aria-label="thumbs up"]')
+    }
+    return behavior
```

Comment:
isn't it possible to create this object just once and not during each rendering phase?

Reply:
@kuzhelov this behavior's wrapper works the same way as we'd use behavior imported from the library, mainly - passing a callback function to component, which is called in `renderComponent.tsx` and passing props into it.